src/objects.js

```diff
@@ -1,41 +1,85 @@
 const createPerson = (name, age) => {
   // your code here
+  return person = {
```

Comment:
Here is a similar issue to previous exercises, where you return an undeclared variable.
Where is person coming from if you don't declare it?

Let's go step by step :)

We can declare a person object in a variable and then return that which would look like this:
`````
const person = {
     name: name,
     age: age,
}

return person
`````

But at this example we don't need to put this into two steps as we don't use person multiple times, only once when we return it, meaning you can just return the object directly like:
`````
return {
     name: name,
     age: age,
}
``````
And then we can even go one step further with the refactoring:
`````
return {
     name,
     age,
}
``````

The properties have the same names as the variables. The object literal property value shorthand was introduced in ES6 to shorten the object initialisation.It allows us to define an object whose keys have the same names as the variables passed in as properties by simply passing the variables. The property value shorthand syntax automatically converts each variable to a key-value pair with the variable name as a property key and the variable value as a property value.